jsapp/js/components/common/koboDropdown.es6

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import autoBind from 'react-autobind';
+import {bem} from 'js/bem';
+import {KEY_CODES} from 'js/constants';
+import './koboDropdown.scss';
+
+export const KOBO_DROPDOWN_THEMES = {};
+new Set([
+  'light',
+  'dark',
+]).forEach((codename) => {KOBO_DROPDOWN_THEMES[codename] = codename;});
+Object.freeze(KOBO_DROPDOWN_THEMES);
+
+export const KOBO_DROPDOWN_PLACEMENTS = {};
+new Set([
+  'up-left',
+  'up-center',
+  'up-right',
+  'down-left',
+  'down-center',
+  'down-right',
+]).forEach((codename) => {KOBO_DROPDOWN_PLACEMENTS[codename] = codename;});
+Object.freeze(KOBO_DROPDOWN_PLACEMENTS);
+
+bem.KoboDropdown = bem.create('kobo-dropdown');
```

Comment:
I like this! On top of the other reasons I find that it gives a good flow for the file and it prepares me for the seeing new components :D

Reply:
I like it too :)